R/plot_ci.R

```diff
@@ -0,0 +1,125 @@
+                                          rep, 
+                                          level = ci_level, 
+                                          seed = ci_random_seed, 
+                                          pass_dist = TRUE)
+  
+  bootstrap_dist <- data.frame(matrix(unlist(sample_stat_dict$dist), 
+                                      nrow = length(sample_stat_dict$dist), 
+                                      byrow = TRUE))
+  
+  colnames(bootstrap_dist) <- c("sample_m")
+  
+  bootstrap_dist_ci <- ggplot2::ggplot(bootstrap_dist, ggplot2::aes(sample_m)) +
+    ggplot2::geom_histogram(fill = "dodgerblue3", 
+                            color = "lightgrey", 
+                            bins = bin_size) +
+    ggplot2::labs(x = x_axis, 
+                  y = y_axis) +
+    ggplot2::theme(text = ggplot2::element_text(size = 16.5)) +
+    ggplot2::geom_vline(xintercept = sample_stat_dict$sample_mean, 
+                        colour = "red", 
+                        size = 1) + 
+    ggplot2::geom_vline(xintercept = unname(sample_stat_dict$lower), 
+                        colour = "purple", 
+                        size = 1, 
+                        linetype = "dashed") +  
+    ggplot2::geom_vline(xintercept = unname(sample_stat_dict$upper), 
+                        colour = "purple", 
+                        size = 1, 
+                        linetype = "dashed")
+  
+  str_1 <- toString(round(sample_stat_dict$sample_mean, 2))
+  str_2 <- toString(round(sample_stat_dict$std_err, 2))
+  
+  annotated_plot <- bootstrap_dist_ci +
+    ggplot2::annotate(geom="label", 
+                      x = sample_stat_dict$sample_mean, 
+                      y = 0.9 * ggplot2::ggplot_build(bootstrap_dist_ci)$layout$panel_params[[1]]$y.range[2],
+                      label = paste(str_1, "+/-", str_2), 
+                      size = 3, 
+                      color = "red",
+                      fill=" lightgrey") + 
+    ggplot2::annotate(geom="label", 
+                      x = unname(sample_stat_dict$lower), 
+                      y = 0.9 * ggplot2::ggplot_build(bootstrap_dist_ci)$layout$panel_params[[1]]$y.range[2], 
+                      label = (toString(round(unname(sample_stat_dict$lower), 2))), 
+                      size = 3, 
+                      color = "purple",
+                      fill=" lightgrey") +
+    ggplot2::annotate(geom="label", 
+                      x = unname(sample_stat_dict$upper), 
+                      y = 0.9 * ggplot2::ggplot_build(bootstrap_dist_ci)$layout$panel_params[[1]]$y.range[2], 
+                      label = (toString(round(unname(sample_stat_dict$upper), 2))), 
+                      size = 3, 
+                      color = "purple",
+                      fill=" lightgrey") +
+    ggplot2::ggtitle(title)
+  
+  ggplot2::ggsave(filename = paste(title, ".png"),
+                  plot = annotated_plot,
+                  path = save_result_to)
+
+  return(annotated_plot)
```

Comment:
```suggestion
annotated_plot
```
I think you don't need `return` here, but please make sure you test it after making the change.